tcms_api/xmlrpc.py

```diff
@@ -123,6 +132,12 @@ def __init__(self, username, password, url):
             else:
                 raise RuntimeError("Unrecognized URL scheme")
 
+        if allow_unverified_ssl:
+            ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
This is modifying the global state of the ssl module which potentially can lead to side effects depending on how that module and the tcms_api module are used by the developer. This could lead to very difficult to debug issues. 

This also affects the SSL module even if for whatever reason the user is connecting to a plain-text endpoint (e.g. reusing the same code base). 

Instead of doing it like this, see the `context` argument of `SafeTransport.__init__()`.